### ISSUE TITLE
Latex error generating report

### ISSUE THREAD

**Issue by dekkers** [minvws/nl-kat-coordination#678]
Pdflatex gives the following error when generating a report:

```[6]
! Missing $ inserted.
<inserted text> 
                $
l.303 ...victim's browser which is treated as a `_
                                                  _Host-` or `__Secure-` coo...

! Missing { inserted.
<to be read again> 
                   _
l.303 ...ictim's browser which is treated as a `__
                                                  Host-` or `__Secure-` cook...

! Missing { inserted.
<to be read again> 
                   _
l.303 ...er which is treated as a `__Host-` or `__
                                                  Secure-` cookie by PHP app...

! Missing } inserted.
<inserted text> 
                }
l.304 
      
! Missing } inserted.
<inserted text> 
                }
l.304 
      
! Missing $ inserted.
<inserted text> 
                $
l.304 
```

**Comment by praseodym** [comment]
Actually the issue is exactly where latex shows, namely underscores in words. [This page lists some solutions], such as escaping underscores or using the underscore package (`\usepackage{underscore}`).

**Comment by praseodym** [comment]
Fixed by #690
